javascript-modal/main.js

```diff
@@ -0,0 +1,16 @@
+var $openBtn = document.querySelector('.modalBtn');
+var $modal = document.querySelector('.modal');
+var $container = document.querySelector('.container');
+
+$openBtn.addEventListener('click', function event() {
+  $container.classList.toggle('active');
+  $modal.style.display = 'flex';
```

Comment:
Instead of doing this by setting the style directly, you should create a class for the modal being on and off, and toggle that class. Setting styling on an element like this is inline styling, which messes with your specificity of the CSS like crazy